### ISSUE TITLE
paddle权重存储格式

### ISSUE THREAD

**Issue by liuhuiCNN** [PaddlePaddle/Paddle#7288]
将tensorflow模型权重转成paddle格式，不太明白paddle权重存储格式，求助。如果知道相关的详细的代码地址麻烦告知。

**Comment by QiJune** [comment]
请参考https://github.com/PaddlePaddle/models/tree/develop/image_classification/tf2paddle

**Comment by lcy-seso** [comment]
请参考PaddlePaddle的文档 F&Q http://www.paddlepaddle.org/docs/develop/documentation/zh/faq/parameter/index_cn.html#id5 有明确的解释和示例。

**Comment by guoshengCS** [comment]
如果是指计算时支持的格式，PaddlePaddle中convolution的权重的shape为[out_channels, in_channels, filter_height, filter_width]，fc的权重的shape为[inut_size, output_size]，batchnorm中各参数的shape为[num_channels, 1]

**Comment by liuhuiCNN** [comment]
@guoshengCS 请问哪些代码可以体现这个顺序？代码的地址可否麻烦告知？谢谢！

**Comment by liuhuiCNN** [comment]
@lcy-seso 我更想知道的是 权重的存储顺序，比如 [5 5 1 20]的一个权重（filter_size=5 in_chanel=1 filter_num=20），存到paddle的tar文件中每个变量的文件中，是怎样的顺序？

**Comment by liuhuiCNN** [comment]
@guoshengCS 好的，谢谢！
总结：
tensorflow中weights [5 5 1 20]转成paddle是 [20 1 5 5]
H W Cin Cout   ----> Cout Cin H W
